prac_08/convert_miles_km.kv

```diff
@@ -0,0 +1,28 @@
+BoxLayout:
+    orientation: 'vertical'
+    BoxLayout:
+        orientation: 'horizontal'
+        TextInput:
+            id: input_distance
+            font_size: 48
+#            on_text: app.handle_calculation(self.text)
+            size_hint_x: 0.75
+        BoxLayout:
+            orientation: 'vertical'
+            size_hint_x: 0.25
+            Button:
+                text: 'Up'
+                on_press: app.handle_increment(input_distance.text, 1)
+            Button:
+                text: 'Down'
+                on_press: app.handle_increment(input_distance.text, -1)
```

Comment:
You could use ids instead of passing variables out

Reply:
Passing variables is better in terms of MVC separation of concerns.